positron/modules/atom_browser_window.js

```diff
@@ -39,6 +40,9 @@ const DEFAULT_WINDOW_FEATURES = [
   'scrollbars',
 ];
 
+// Map from nsIDOMWindow instances to BrowserWindow instances.
+let browserWindows = new Map();
```

Comment:
Similarly, just "map of DOM Window instances to BrowserWindow intsances" probably.